src/routes/docs/tutorials/flutter-auth/step-2/+page.markdoc

```diff
@@ -0,0 +1,51 @@
+---
+layout: tutorial
+title: Create project
+description: Add Authentication to a Flutter project using Appwrite.
+step: 2
+---
+
+
+You can create a Flutter project using the following command
+
+```sh
+flutter create my_flutter_project
+```
+
+The command will automatically create a Flutter project with the name `my_flutter_project`. You can change the name of the project by changing the name of the project in the command.
+
+You will see the following output after the project is created.
+
+```sh
+Creating project my_flutter_project...
```

Comment:
I love this, but maybe it's excessively long. Idk if this is overly helpful.

Up to you.